spring-framework/spring-framework-quiz.md

```diff
@@ -847,13 +847,6 @@ Explanation: Here only one line can throw NPE. Calling getClass() from context.g
 - [ ] During startup, the SpringFactoryInitializr collects all files in the CONFIG-INF directory from each dependency and downloads binaries to run each file.
 - [ ] During startup and shutdown, the SpringFactoryInitializr downloads project configs for all configured dependencies.
 
-#### Q82. What methods does this Pointcut expression reference?
```

Comment:
This question was missing the code snippet but idk I also think it's answer was wrong

Reply:
This example in the official documentation (but older version) seems comparable, so I would expect it to be this answer: https://docs.spring.io/spring-framework/docs/2.0.x/reference/aop.html#:~:text=the%20execution%20of%20any%20method%20defined%20in%20the%20service%20package%20or%20a%20sub%2Dpackage%3A

And a (nearly) matching example in this more recent tutorial: https://www.baeldung.com/spring-aop-pointcut-tutorial#1-execution

Reply:
Yeah I agree with all that! So the updated answer ("with subpackages") is below. 

You think I should add those links in an explanation or we are good?

Reply:
Thank you! Either is fine for me, but I'm not a maintainer or anything